web/src/lib/roadmap/buildUserRoadmap.test.ts

```diff
@@ -462,6 +488,34 @@ describe("buildUserRoadmap", () => {
         expect(getLastCalledWith(mockRoadmapBuilder)[0].addOns).not.toContain("cannabis-conditional");
       });
     });
+
+    describe("non essential questions", () => {
+      it("adds addOn if nonEssentialsQuestion value is true", async () => {
+        await buildUserRoadmap(
+          generateStartingProfile({
+            industryId: "test-industry",
+            nonEssentialQuestions: {
+              "test-question-1": true,
+              "test-question-2": false,
+            },
+          })
+        );
+        expect(getLastCalledWith(mockRoadmapBuilder)[0].addOns).toContain("test-addOn-1");
+      });
+
+      it("doesn't add addOn if nonEssentialsQuestion is value is false", async () => {
+        await buildUserRoadmap(
+          generateStartingProfile({
+            industryId: "test-industry",
+            nonEssentialQuestions: {
+              "test-question-1": true,
+              "test-question-2": false,
+            },
```

Comment:
[dust] This could be one test I think with just both assertions

Reply:
I was thinking of that as well, but I remember discussing with @aloverso at some point that we shouldn't have an `and` in a test because then you are testing for two different behaviors and rewriting this would require an `and`? I don't have strong preference, could go either way.

Reply:
@somabadri @adidner I do stand by that rule of thumb. In this case, if you *wanted* you could probably name the test as something like "only adds addOn when nonEssentialsQuestion value is true" and the "only" implies the not-add-if-false behavior, and then you could put both assertions in the same test without it feeling like it's two separate "and" tests. I don't have a strong preference - I think simplicity and clarity are the important things to optimize for, and either option is fairly simple so so strong preference.